engine/printer/__init__.py

```diff
@@ -86,26 +84,20 @@ def _get_style_from_string(content: str) -> str:
     else:
         return ""
 
-    
+
 def _get_input_options(option_strings: list[str]) -> dict:
     options_map = {}
-
-    for option_string in option_strings:
-        for index, char in enumerate(option_string):
-            shortcut = str(index + 1)
-
-            if not options_map.get(shortcut):
-                options_map[shortcut] = {
```

Comment:
Before, we'd only register the shortcut if len(option_string) was greater than the number of options shortcuts already registered.  So we couldn't accuse Dee 😭